src/algokit/cli/init.py

```diff
@@ -165,55 +165,38 @@ def init_command(
     open_ide: bool,
 ) -> None:
     """Initializes a new project from a template."""
-    # copier is lazy imported for two reasons
-    # 1. it is slow to import on first execution after installing
-    # 2. the import fails if git is not installed
 
-    try:
-        # TODO: copier is typed, need to figure out how to force mypy to accept that or submit a PR
-        #       to their repo to include py.typed file
-        import copier  # type: ignore[import]
-    except ImportError as ex:
+    if not shutil.which("git"):
```

Comment:
I've checked the implementation of `which` that `plumbum` uses (ie what copier uses under the hood). It's a very old library, dating back to at least Python 2.5, so they don't use the built-in `shutil.which` that was added in 3.3, but the implementations are essentially equivalent. So if this check passes, we can be confident that copier will import successfully.